srcpkgs/booster/template

```diff
@@ -0,0 +1,48 @@
+# Template file for 'booster'
+pkgname=booster
+version=0.7
+revision=1
+build_style=go
+go_import_path=github.com/anatol/booster
+go_package="${go_import_path}/generator ${go_import_path}/init"
+hostmakedepends="git ruby-ronn"
+depends="busybox-static"
+short_desc="Fast and secure initramfs generator"
+maintainer="travankor <travankor@tuta.io>"
+license="MIT"
+homepage="https://github.com/anatol/booster"
+distfiles="https://github.com/anatol/booster/archive/${version}.tar.gz"
+checksum=70cf535865e8f528a233fd4ca04eb178b11a897f1283877f63e9056392475f3a
+conf_files="/etc/booster.yaml"
+
+pre_build() {
+	export CGO_ENABLED=0
```

Comment:
Could you please move `CGO_ENABLED=0` to `go install` of the init binary at the line 27?

Reply:
`ldd /usr/lib/booster/init` not a dynamic executable

I think `init` is already static?

Reply:
What I meant `CGO_ENABLED=0` should be used for `init` only. It probably be harmful for `generator` binary.

Reply:
I see, thanks.